### DISCUSSION TITLE
Validating Admission Webhook --namespaced

### DISCUSSION THREAD

**Discussion by tomkennes** [argoproj/argo-events#1481] [Q&A]
Hi there,

First of all, thank you all for the awesome work you are putting into the Argo products! I am a fan! :) 

Just a short question, is there a reason why the Validating Admission Webhook (https://argoproj.github.io/argo-events/validating-admission-webhook/) is only available on a cluster-level and not in the namespaced version as are most other argo-events components? 

Thanks, kind regards,
Tom

**Comment by whynowy** [comment]
Thanks!

Could you raise an issue for that? A PR to implement that would be even better :)

https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-namespaceselector

**Reply by tomkennes** [reply]
Thanks! I haven't contributed to Argo Events yet, so I am not too familiar with the code-base, but I will give it a shot! :) Otherwise, I'll raise an issue.

**Reply by tkennes** [reply]
@whynowy, I wasn't sure whether I would able to do it, but I think I managed to do it! :) Please, let me know whether there is anything missing: https://github.com/argoproj/argo-events/pull/1483

**Reply by tkennes** [reply]
Edit: the pr is: https://github.com/argoproj/argo-events/pull/1484
